src/me.rs

```diff
@@ -315,9 +329,14 @@ fn get_subset_predictors<T: Pixel>(
       tile_me_stats[tile_bo.0.y - 1][tile_bo.0.x + clipped_half_w],
     ));
   }
+
+  // I tried sampling far right and far bottom edges, but this had worse results
```

Comment:
Hi,
It may be better if the documentation is in third-person or more of technical one instead of being in first person stating as I... etc